### PR TITLE
Update docs of assets-registry

### PR DESCRIPTION
**Pull request by andor0**
## Issue
https://app.clickup.com/t/28n4xye

### CODE REVIEW

File: frame/assets-registry/src/lib.rs [hunk 1]
```diff
@@ -156,10 +174,10 @@ pub mod pallet {
 		local_admin: Option<T::AccountId>,
 		foreign_admin: Option<T::AccountId>,
 		// TODO: split this into 2 pairs
-		// 1. (xcm location -> local asset id as used in our tuntime), so that when others send our
-		// id to our chain we can trust them 2. (local qasset id - > remote location -> remote
-		// asset id) so then when we send our local asset to remote chain we know what id we should
-		// envode.
+		// 1. (xcm location -> local asset id as used in our runtime), so that when others send our
+		// id to our chain we can trust them
+		// 2. (local qasset id -> remote location -> remote asset id), so then when we send our
```
Comment (hussein-aitlahcen) [inline frame/assets-registry/src/lib.rs:179]: 'asset'

File: frame/assets-registry/src/lib.rs [hunk 2]
```diff
@@ -196,26 +214,44 @@ pub mod pallet {
...
 
 	#[pallet::error]
 	pub enum Error<T> {
+		/// Only local admin or foreign admin can do this.
 		OnlyAllowedForAdmins,
+		/// The lolal asset Id already used.
 		LocalAssetIdAlreadyUsed,
+		/// The foreign asset Id already used.
 		ForeignAssetIdAlreadyUsed,
+		/// The lolal asset Id not found.
```
Comment (hussein-aitlahcen) [inline frame/assets-registry/src/lib.rs:238]: 'local'

File: frame/assets-registry/src/lib.rs [hunk 3]
```diff
@@ -196,26 +214,44 @@ pub mod pallet {
...
 		},
+		/// Local admin or foreign admin updated a metadata of foreign asset.
 		AssetMetadataUpdated(T::LocalAssetId),
 	}
 
 	#[pallet::error]
 	pub enum Error<T> {
+		/// Only local admin or foreign admin can do this.
 		OnlyAllowedForAdmins,
+		/// The lolal asset Id already used.
```
Comment (hussein-aitlahcen) [inline frame/assets-registry/src/lib.rs:234]: 'local'

File: frame/assets-registry/src/lib.rs (hunk 1 above)
Comment (andor0) [reply frame/assets-registry/src/lib.rs]: fixed

File: frame/assets-registry/src/lib.rs (hunk 3 above)
Comment (andor0) [reply frame/assets-registry/src/lib.rs]: fixed

File: frame/assets-registry/src/lib.rs (hunk 2 above)
Comment (andor0) [reply frame/assets-registry/src/lib.rs]: fixed

**Review by dzmitry-lahoda — APPROVED** [review]

**Review by PoisonPhang — APPROVED** [review]

**Pull request merged**